show/main.py

```diff
@@ -796,6 +810,68 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+#
+# 'breakout' group ###
+#
+@interfaces.group(invoke_without_command=True)
+@click.pass_context
+def breakout(ctx):
+    """Show interface breakout"""
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {'db': config_db}
+
+    curBrkout_tbl = config_db.get_table('BREAKOUT_CFG')
```

Comment:
We need gracefully handle the cases where BREAKOUT_CFG is not in configDB,  platform.json/hwsku.json are not available cases.

Reply:
done